### ISSUE TITLE
Unsubscribe from all active subscriptions

### ISSUE THREAD

**Issue by bachwehbi** [beebotte/bbt_node#23]
Instead of unsubscribing from every single active subscription, it would be useful to have an `unsubscribeAll` method.

```
//unsubscribes from all active subscriptions
unsubscribeAll() 

//unsubscribes from all active subscriptions on a given channel
unsubscribeAll(someChannel)
```

**Comment by bachwehbi** [comment]
This applies to `socketio` and `mqtt` transports (`streaming` API)

**Comment by bachwehbi** [comment]
Fixed by 03f2e2c
